docs/rmlint.1.rst

```diff
@@ -816,19 +816,19 @@ OTHER STAND-ALONE COMMANDS
     snapshots (requires root).
 
 :``rmlint --is-reflink [-v|-V] <file1> <file2>``:
-    Tests whether ``file1`` and ``file2`` are reflinks (reference same data).
+    Tests whether ``file1`` and ``file2`` are reflinks (they reference the same data).
     This command makes ``rmlint`` exit with one of the following exit codes:
 
     * 0: files are reflinks
     * 1: files are not reflinks
     * 3: not a regular file
     * 4: file sizes differ
     * 5: fiemaps can't be read
-    * 6: file1 and file2 are the same path
+    * 6: file1 and file2 are at the same path
```

Comment:
I'm not sure about this change. This status is for when the two paths provided are literally the same path after resolving symlinks, hence no data needs to be looked at.

Reply:
Maybe "path1" and "path2" are more suitable argument names. Then this could read:
>6: path1 and path2 point to the same file (after resolving symlinks)

Reply:
Well, "the same file" is ambiguous here, because it could be taken to mean "the same inode." But that's a hardlink - two different paths that point to the same file. The paths themselves are equal after resolving symlinks in this case.